opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/README.adoc

```diff
@@ -151,11 +151,6 @@ org.opennms.web.rest.support.SearchPropertiesToAsciidocTest
 | assetRecord.vendorAssetNumber | String | Vendor Asset Number
 | assetRecord.vendorFax | String | Vendor Fax
 | assetRecord.vendorPhone | String | Vendor Phone
-| assetRecord.vmwareManagedEntityType | String | VMware Managed Entity Type
```

Comment:
Please add a note to 'breaking changes' as this alters the REST endpoint.

Reply:
Good point.